src/20_Components/amp-bind.html

```diff
@@ -91,6 +97,13 @@
   </amp-img>
 
   <!--
+  The src of the video will change when the `currentAnimal` variable
```

Comment:
Won't it also change when allAnimals changes as explained above? (I know you copied it from below, just noticed the inconsistency).